### ISSUE TITLE
InvalidArgumentError (see above for traceback): Default AvgPoolingOp only supports NHWC.

### ISSUE THREAD

**Issue by miaozhang0525** [renqianluo/NAO#10]
I followed your Requirment and Dependency Tensorflow == 1.4.0, Pytorch == 0.3.1, But I got follow error:

InvalidArgumentError (see above for traceback): Default AvgPoolingOp only supports NHWC.
	 [[Node: child_1/layer_0/cell_4/x_conv/average_pooling2d/AvgPool = AvgPool[T=DT_FLOAT, data_format="NCHW", ksize=[1, 1, 3, 3], padding="SAME", strides=[1, 1, 1, 1], _device="/job:localhost/replica:0/task:0/device:CPU:0"](child_1/layer_0/layer_base/bn/Identity)]]

Am I wrong for the configuration?

I also tried on tensorflow-gpu, it either does not work.

**Comment by miaozhang0525** [comment]
Sometimes is IMaxPoolingOp only supports NHWC.

**Comment by mcbearone** [comment]
Hello! Have you solved this problem? I meet the same problem when i run "bash train_search.sh"

**Comment by mcbearone** [comment]
> Sometimes is IMaxPoolingOp only supports NHWC.

Hello! Have you solved this problem? I meet the same problem when i run "bash train_search.sh"

**Comment by mcbearone** [comment]
> I followed your Requirment and Dependency Tensorflow == 1.4.0, Pytorch == 0.3.1, But I got follow error:
> 
> InvalidArgumentError (see above for traceback): Default AvgPoolingOp only supports NHWC. [[Node: child_1/layer_0/cell_4/x_conv/average_pooling2d/AvgPool = AvgPool[T=DT_FLOAT, data_format="NCHW", ksize=[1, 1, 3, 3], padding="SAME", strides=[1, 1, 1, 1], _device="/job:localhost/replica:0/task:0/device:CPU:0"](child_1/layer_0/layer_base/bn/Identity)]]
> 
> Am I wrong for the configuration?
> 
> I also tried on tensorflow-gpu, it either does not work.

Hello! Have you solved this problem? I meet the same problem when i run "bash train_search.sh"
